test/index.spec.ts

```diff
@@ -33,11 +34,6 @@ describe('Unit testing for typedoc-plugin-versions', function () {
 				'did not provided a correctly formatted patch version'
 			);
 		});
-		it('throws error if version not defined', function () {
```

Comment:
Let's keep this test in place.

Reply:
Resolved by 148563f91a206b97b60f4642f2dd985b0acced72